router/pkg/config/config.schema.json

```diff
@@ -19,6 +19,12 @@
         "token": {
           "type": "string",
           "description": "The token used to authenticate with other component from Cosmo. Can be ommitted if the router is started with a static execution config."
+        },
+        "sign_key": {
```

Comment:
how about "config_signature_key" and adding "config_signature_algorithm" with an enum value, defaulting to "hmac-sha-256" to allow for changes in the future?

Reply:
Let's wait for feedback. We can always make it flexible while preserving the defauilt.